README.md

```diff
@@ -1 +1,33 @@
-# node-global-lab
\ No newline at end of file
+# Use REPL and Global
+
+## Objectives
+
+1. Use REPL
+1. Access global
+1. Understand scoping and explicit vs. implicit declaration
+
+## Introduction
+
+Understanding global and scoping will give you a good foundation for future development.
```

Comment:
Make intro more catchy - you are going to be using the REPL to create bank user and bank account objects. etc. etc.